src/components/Input/README.md

```diff
@@ -26,18 +26,19 @@ const InputWrapper = () => {
-| success      | boolean                        | Yes      | None      | To enable success state                               |
-| label        | string \| ReactNode            | Yes      | None      | content to be shown above the input as a label        |
-| notification | string \| ReactNode            | Yes      | None      | content to be shown below the input as a notification |
-| onChange     | Function                       | No       | None      | onChange handler                                      |
-| addonAfter   | string \| ReactNode            | Yes      | None      | to show after input                                   |
-| addonBefore  | string \| ReactNode            | Yes      | None      | to show before input                                  |
-| inputSize    | "default" \| "small" \| "lage" | Yes      | "default" | the size of the inputbutton                           |
-| disabled     | boolean                        | Yes      | None      | To disable the input                                  |
+| Prop name      | Type                                                          | Optional | Default   | Description                                           |
+| -------------- | ------------------------------------------------------------- | -------- | --------- | ----------------------------------------------------- |
+| placeholder    | string                                                        | Yes      | None      | This is a placeholder description                     |
+| value          | string                                                        | Yes      | None      | This is a value of the input                          |
+| state          | "success" \| "warning" \| "error"                             | Yes      | None      | To enable state of input (success, warning or error)  |
+| label          | string \| ReactNode                                           | Yes      | None      | content to be shown above the input as a label        |
+| notification   | string \| ReactNode                                           | Yes      | None      | content to be shown below the input as a notification |
+| onChange       | Function                                                      | Yes      | None      | onChange handler                                      |
+| addonAfter     | string \| ReactNode                                           | Yes      | None      | to show after input                                   |
+| addonBefore    | string \| ReactNode                                           | Yes      | None      | to show before input                                  |
+| inputSize      | "default" \| "small" \| "lage"                                | Yes      | "default" | the size of the inputbutton                           |
+| type           | "default" \| "search" \| "number" \| "password" \| "textarea" | Yes      | "default" | Type of input field to be displayed                   |
```

Comment:
please mentions we are not supporting all the html input types value, so it clearly indicate that only support some html input type.